package.json

```diff
@@ -22,6 +22,7 @@
   "scripts": {
     "coveralls": "cat ./coverage/lcov.info | node node_modules/.bin/coveralls",
     "debug:integration": "node --inspect-brk ./test/run-test.js",
+    "debug:test": "node --inspect-brk ./node_modules/.bin/jest --runInBand --testNamePattern",
```

Comment:
this seems unrelated to the PR?!

Reply:
Yep, sorry my bad, it was meant to be in a separate PR, will remove it.